package-lock.json

```diff
@@ -36478,7 +36478,7 @@
     },
     "packages/next-steps": {
       "name": "@hashicorp/react-next-steps",
-      "version": "0.3.0",
+      "version": "0.4.0",
```

Comment:
Unintended change?

Reply:
Hmmm...it occurred after I ran `npm i` to get installed deps up to date 🤔 I can revert before merge though

Reply:
ah ok, its in the lock file so I guess that should be ok!